packages/theming/atlas/src/themesource/atlas_ui_resources/web/_variables.scss

```diff
@@ -393,6 +393,8 @@ $tabs-color: $font-color-detail !default;
 $tabs-color-active: $font-color-default !default;
 $tabs-lined-color-active: $font-color-default !default;
 
+$tabs-lined-border-width: 3px !default;
```

Comment:
Please also add the new variable inside `widgets-resources/packages/theming/atlas/src/theme/web/custom-variables.scss` 🙂

Reply:
Added to custom variables